pkg/api/validation/validation.go

```diff
@@ -2346,6 +2356,7 @@ func ValidateResourceRequirements(requirements *api.ResourceRequirements, fldPat
 		if api.IsStandardResourceName(string(resourceName)) {
 			allErrs = append(allErrs, validateBasicResource(quantity, fldPath.Key(string(resourceName)))...)
 		}
+		allErrs = append(allErrs, validateContainerResourceLimits(string(resourceName), quantity, fldPath)...)
```

Comment:
I don't think this is right.

I agree we should check that you can't have a quota on limits.cpu that is < 10m, but this is not that...
I also think we should check in LimitRange validation that max[cpu] and default[cpu] cannot be < 10m.